### ISSUE TITLE
Translation to other languages

### ISSUE THREAD

**Issue by g200kg** [WICG/serial#117]
If I want to translate this specification into a language other than English and publish it, is there any procedure I should take?

**Comment by reillyeon** [comment]
Yes, please follow the guidelines here: https://www.w3.org/Consortium/Translation/Overview.html

The translation should be contributed back to this repository.

**Comment by g200kg** [comment]
Thank you!
